app/controllers/users_controller.rb

```diff
@@ -116,8 +116,7 @@ def profile
                        .page(params[:page])
                        .order("nid DESC")
                        .where(status: 1, uid: @user.uid)
-    @questions = DrupalNode.questions
-                           .where(status: 1, uid: @user.uid)
+    @questions = @user.user.questions
```

Comment:
I think that's not it. I tried that but when I ran `rake test:all`a lot of errors like that popped out:

```NoMethodError: undefined method `questions' for #<DrupalUsers:0x00555a52b4a828>```